src/types/aegisAI.ts

```diff
@@ -43,6 +43,14 @@ export type AegisAIValidationErrorType = {
   type: string;
 };
 
+// AEGIS AI Response Types
+export type AegisAISuggestCweResponseType = {
```

Comment:
I think is the same as `CweSuggestionDetails` right? we could remove the other one